### PR TITLE
Issue #3404574: Reorganize filters on admin/people page

### PR DESCRIPTION
**Pull request by VBaldych**
## Problem
Currently, the admin/people overview has a lot of filters and we have added them not in the best way.
It is hard to even find the user's name which should be the main element there.

## Solution
- Implement new design, to reorganize filter structure in admin/people page.
- Reorganize filters in `views.view.user_admin_people` using module https://www.drupal.org/project/views_ef_fieldset
- Avoid using form_alter

## Issue tracker
- https://getopensocial.atlassian.net/browse/PROD-23418
- https://www.drupal.org/project/social/issues/3404574
- https://github.com/goalgorilla/cablecar/pull/973/
- [new design](https://www.figma.com/file/J2sPkAuodbyaVK2Ztq1OJ3/Jira-tickets?type=design&node-id=2142%3A9163&mode=design&t=mILXRhhTt7fSHDa2-1)

## Theme issue tracker
n/a

## How to test
- [ ] Login as admin
- [ ] Enable social_follow_tag
- [ ] Go to admin/people page
- [ ] You should see "main" container container with fields: Name/Email, Role, Group, Status, Permission (if logged in as uid=1)
- [ ] You should see "Show more/less" button. Click on it
- [ ] You should see "secondary" container with fields: Registration date, Last activity, Content tags, Permission

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Screenshots
image
image
image
image

## Release notes
Before, the **_admin/people_** overview page had a lot of filters, added not in the best way.

Now we re-organized filters according to a new improved design.

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

File: modules/social_features/social_user/src/Plugin/better_exposed_filters/filter/Html5Date.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Drupal\social_user\Plugin\better_exposed_filters\filter;
+
+use Drupal\better_exposed_filters\Plugin\better_exposed_filters\filter\FilterWidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * HTML5 date widget implementation.
```
Comment (ribel) [inline modules/social_features/social_user/src/Plugin/better_exposed_filters/filter/Html5Date.php:9]: @VBaldych could you explain why this plugin is needed, where it's used and what you take as a base for it?

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

File: modules/social_features/social_user/src/Plugin/better_exposed_filters/filter/Html5Date.php (hunk 1 above)
Comment (VBaldych) [reply modules/social_features/social_user/src/Plugin/better_exposed_filters/filter/Html5Date.php:9]: The main idea to create this filter widget is avoid using **form_alter** with hardcode for Date filters. In our case, we use it on 'Registered' and 'Last active' fields

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3664-srkuritee3e04ov7pkzozse2orfybunv.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/658bf9e7dd3e0563dfee25f7

**Comment by ribel** [comment]
Changes were tested and worked correctly on new and existing installations.
🍒 picked to the 12.1.x branch.

**Pull request merged**